hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecordPayload.java

```diff
@@ -217,4 +217,16 @@ static Option<String> getPayloadClassNameIfPresent(Properties props) {
     // Need to transparently change to org.apache.hudi.
     return Option.ofNullable(payloadClassName).map(className -> className.replace("com.uber.hoodie", "org.apache.hudi"));
   }
+
+  /**
+   * Returns the writer payload override class if present in the properties.
+   * @param properties the properties to inspect
+   * @return an Option containing the writer payload override class name if present, otherwise an empty Option
+   */
+  static Option<String> getWriterPayloadOverride(Properties properties) {
+    if (properties.containsKey("hoodie.datasource.write.payload.class")) {
```

Comment:
`if (containsKey()..) {Option.of()} else {}` -> `Option.ofNullable()` ?

Reply:
Thanks! I'm realizing I can also just do `Option.ofNullable` on the result of the `get`